dnf/cli/output.py

```diff
@@ -1647,7 +1641,49 @@ def historyListCmd(self, tids):
                 rmark = '<'
             if transaction.altered_gt_rpmdb:
                 lmark = '>'
-            print(fmt % (transaction.tid, name, tm, uiacts, num), "%s%s" % (lmark, rmark))
+
+            t_tm = tm
+            t_id = transaction.tid
+            t_name = name
+            t_uiacts = uiacts
+            t_rmark = "%s%s" % (lmark, rmark)
+            columns.append(
+                (t_id, t_tm, t_name, t_uiacts, t_rmark)
+            )
+
+        # I want to fix the size of columns except for name->Command Line (2nd column)
+        # I need to find the longest line
+        # name length at least 16
+        nm_len = max([16] + [len(x[2]) for x in columns])
+        id_len = 4
+        tm_len = 16
+        ui_len = 14
+        rm_len = 7
+        separator = " | "
+        sp_len = len(separator)
+
+        if self.term.real_columns is not None:
+            nm_len = self.term.real_columns - (id_len + tm_len + ui_len + rm_len) - sp_len * 4
+            title_line = "-" * self.term.real_columns
+        else:
+            title_line = "-" * (nm_len + id_len + tm_len + ui_len + rm_len + sp_len * 4)
+
+        print(fill_exact_width(_(" ID"), id_len, id_len),
```

Comment:
I belive that the calculation of id_len is invalid because it is only valid for English. In other languages `_(" ID")` will be replaced by translation.